from typing import Callable


def find_root(func: Callable[[float], float], interval: tuple[float, float], precision: float) -> float:
	raise Exception
